runtests.sh

```diff
@@ -341,6 +344,22 @@ compile_cpp
 # unconditionally report on the state of monai
 print_version
 
+if [ $doCopyRight = true ]
+then
+    # check copyright headers
+    find "$(pwd)/monai" "$(pwd)/tests" -type f \
+      ! -wholename "*_version.py" -and -name "*.py" -or -name "*.cpp" -or -name "*.cu" -or -name "*.h" |\
+      while read -r fname; do
+        if ! grep "http://www.apache.org/licenses/LICENSE-2.0" "$fname" > /dev/null; then
+            print_error_msg "Missing the license header in file: $fname"
+            echo "Please add the licensing header to the file."
+            echo "  See also: https://github.com/Project-MONAI/MONAI/blob/dev/CONTRIBUTING.md#checking-the-coding-style"
+            echo ""
+            exit 1
```

Comment:
I think better to continue the check and print all the missing files instead of `exit 1`.

Thanks.

Reply:
yes, now it will finish the loop and print the final results